### ISSUE TITLE
Readme Request

### ISSUE THREAD

**Issue by PowerPress** [Coalfire-Research/DeathMetal#2]
Awesome project please put a link to your blog in the readme for more info https://www.coalfire.com/The-Coalfire-Blog/April-2019/The-Death-Metal-Suite

**Comment by panch0** [comment]
Good idea! Will do later tonight - have other pressing things to get done during the day :/
